client/stylesheets/_toCat.scss

```diff
@@ -5401,6 +5413,10 @@ On click the images will be displayed at normal size to complete the effect
 	margin-right: 12px;
 }
 
+ul button a {
```

Comment:
Merge rule `ul button a` with rule on line 604<br>Selector should have depth of applicability no greater than 2, but was 3